ci-operator/step-registry/openshift-qe/installer/azure/ipi/ovn/multi-ipsec/openshift-qe-installer-azure-ipi-ovn-multi-ipsec-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
```

Comment:
@shahsahil264 This files looks exactly same as https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/installer/azure/ipi/ovn/ipsec/openshift-qe-installer-azure-ipi-ovn-ipsec-workflow.yaml now. So I don't think we need this new workflow and the multi-ipsec folder.

Reply:
Yes, I was just testing out some things, I'll delete it. Thanks!